templates/applications/export.html

```diff
@@ -10,18 +10,18 @@
         {{ helper.render_app_local_nav(app, "tasks", current_user) }}
     </div>
     <div id="body" class="col-md-9" style="min-height:400px;">
-        {{ helper.render_app_title(app, subtitle=_('Export All Tasks and Task Runs'))}}
+        {{ helper.render_app_title(app, upload_method, subtitle=_('Export All Tasks and Task Runs'))}}
         <p>{{ _('You can export and download') }} <strong>{{ _('all the available') }}</strong> {{ _('Tasks and Task Runs (the submitted answers by the users) in CSV or JSON formats.') }}</p>
         <div class="row">
             <div id="csv" class="col-md-5 well">
-                <h2>Export in CSV format</h2>
-                <a href={{url_for('app.export_to', short_name=app.short_name, type='task', format='csv')}} class="btn btn-lg btn-primary">{{ _('Tasks') }}</a>
-                <a href={{url_for('app.export_to', short_name=app.short_name, type='task_run', format='csv')}} class="btn btn-lg btn-primary">{{ _('Task Runs') }}</a>
+                <h2>{{_('Export in CSV format')}}</h2>
+                <a href={{url_for('app.export_to', short_name=app.short_name, type='task', format='csv')}} rel="nofollow" class="btn btn-lg btn-primary" download>{{ _('Tasks') }}</a>
```

Comment:
It is on our default theme. I guess @therealmarv worked on it (I may be wrong, though) and is for web crawlers related stuff or something similar